docs/docs/faq.mdx

```diff
@@ -74,8 +74,8 @@ While we're working on improving the efficiency, there are a couple of tricks th
    This will allow you to continuously see what have been computed so far.
    Refresh the browser once in a while when new metrics are done computing in your first terminal.
 
-2. You can also kill the import process as soon as metrics have started to compute. This will leave you with a project containing fewer metrics. As a consequence, you will not be able to see as many insights as if the process is allowed to finish.
-   However, you can always use the [`encord-active metric run`](./cli#run) command to run metrics that are missing.
+2. You can also kill the import process as soon as metrics have started to compute. This will leave you with a project containing less metrics. As a consequence, you will not be able to see as many insights as if the process is allowed to finish.
```

Comment:
```suggestion
2. You can also kill the import process as soon as metrics have started to compute. This will leave you with a project containing fewer quality metrics. As a consequence, you will not be able to see as many insights as if the process is allowed to finish.
```